tests/src/JIT/HardwareIntrinsics/X86/Avx/Add.Double.cs

```diff
@@ -28,32 +28,41 @@ private static void AddDouble()
                 // Validates basic functionality works, using Unsafe.Read
                 test.RunBasicScenario_UnsafeRead();
 
```

Comment:
The majority of these test changes are the same (since they are all generated from the template). I've left a few comments on each of the changes below.